src/main/webapp/app/entities/saturn-vault/saturn-vault.controller.js

```diff
@@ -47,6 +51,25 @@
             }
         }
 
+        function clickToCopy(passToCopy) {
+			var body = angular.element(document.body);
+			var textarea = angular.element('<textarea/>');
+			textarea.css({
+				display: 'hidden',
+				position: 'fixed',
+				opacity: '0'
+			});
+
+			textarea.val(passToCopy);
+			body.append(textarea);
+			textarea[0].select();
+			if (document.execCommand('copy')) {
+                AlertService.success(vm.copySuccessText)
+			} else {
+                AlertService.error(vm.copyErrorText);
+            }
+		}
+
```

Comment:
Can we take a minute to discuss our lord and savior, the format button in VSCode? Real though I ain't approving this till the tabs are fixed